### PR TITLE
Reorder zsh because newer macOS uses zsh

### PR DESCRIPTION
**Pull request by karlhorky**
First of all, thanks for React Native! 🙌 Such amazing software and great DX

Just a quick PR to reorder the zsh and bash files, since the newer versions of macOS have switched from bash to zsh (so this will become increasingly common that people will need to use the zsh files)

### CODE REVIEW

File: website/versioned_docs/version-0.70/_getting-started-macos-android.md [hunk 1]
```diff
@@ -73,17 +73,15 @@ Finally, click "Apply" to download and install the Android SDK and related build
 
 The React Native tools require some environment variables to be set up in order to build apps with native code.
 
-Add the following lines to your `$HOME/.bash_profile` or `$HOME/.bashrc` (if you are using `zsh` then `~/.zprofile` or `~/.zshrc`) config file:
+Add the following lines to your `$HOME/.zprofile` or `$HOME/.zshrc` (if you are using `bash`, then `~/.bash_profile` or `~/.bashrc`) config file:
```
Comment (cortinico) [inline website/versioned_docs/version-0.70/_getting-started-macos-android.md:76]:
```suggestion
Add the following lines to your `~/.zprofile` or `~/.zshrc` (if you are using `bash`, then `~/.bash_profile` or `~/.bashrc`) config file:
```

File: website/versioned_docs/version-0.70/_getting-started-macos-android.md [hunk 2]
```diff
@@ -73,17 +73,15 @@ Finally, click "Apply" to download and install the Android SDK and related build
...
 ```shell
 export ANDROID_SDK_ROOT=$HOME/Library/Android/sdk
 export PATH=$PATH:$ANDROID_SDK_ROOT/emulator
 export PATH=$PATH:$ANDROID_SDK_ROOT/platform-tools
 ```
 
-> `.bash_profile` is specific to `bash`. If you're using another shell, you will need to edit the appropriate shell-specific config file.
-
-Type `source $HOME/.bash_profile` for `bash` or `source $HOME/.zprofile` to load the config into your current shell. Verify that ANDROID_SDK_ROOT has been set by running `echo $ANDROID_SDK_ROOT` and the appropriate directories have been added to your path by running `echo $PATH`.
+Run `source $HOME/.zprofile` (or `source $HOME/.bash_profile` for `bash`) to load the config into your current shell. Verify that ANDROID_SDK_ROOT has been set by running `echo $ANDROID_SDK_ROOT` and the appropriate directories have been added to your path by running `echo $PATH`.
```
Comment (cortinico) [inline website/versioned_docs/version-0.70/_getting-started-macos-android.md:84]: Same as above, let's use `~` here

**Review by cortinico — CHANGES_REQUESTED** [review]
Seems like you're editing the docs for 0.70 while you should update the docs inside the `./docs` folder

File: website/versioned_docs/version-0.70/_getting-started-macos-android.md (hunk 2 above)
Comment (karlhorky) [reply website/versioned_docs/version-0.70/_getting-started-macos-android.md]: done in caa1619e2b511e0cd6f1d2c95bcd19cb2e698915

**Comment by karlhorky** [comment]
> Seems like you're editing the docs for 0.70 while you should update the docs inside the `./docs` folder

Ah sorry, updated the location now

**Comment by karlhorky** [comment]
Glad to help :) Thanks for the merge!

**Pull request merged**